lgc/test/ElfRelocationAndNote.lgc

```diff
@@ -2,16 +2,16 @@
 ; is no relocation. It also checks the ISA name is correctly generated in the
 ; .note section.
 
-; RUN: lgc -mcpu=gfx802 -extract=2 -filetype=obj -o %t.vs.elf %s
-; RUN: lgc -mcpu=gfx802 -extract=3 -filetype=obj -o %t.fs.elf %s
-; RUN: lgc -mcpu=gfx802 -extract=1 -l -o %t.pipe.elf %s %t.vs.elf %t.fs.elf
+; RUN: lgc -mcpu=gfx1010 -extract=2 -filetype=obj -o %t.vs.elf %s
+; RUN: lgc -mcpu=gfx1010 -extract=3 -filetype=obj -o %t.fs.elf %s
+; RUN: lgc -mcpu=gfx1010 -extract=1 -l -o %t.pipe.elf %s %t.vs.elf %t.fs.elf
```

Comment:
Note: This case is different, since the RUN lines aren't actually identical.

Reply:
Thank you.